ad9371/sw/headless.c

```diff
@@ -127,7 +127,7 @@ dac_core  ad9371_tx_core_init = {
 *******************************************************************************/
 int main(void)
 {
-	ADI_ERR				ad9528Error;
```

Comment:
why not leave `ad9528Error` here ?
changing it seems a bit un-necessary when fixing the leak